http_response_header_optional_fields.md

```diff
@@ -1,61 +0,0 @@
-This document describes a set of rules which may be used to determine which optional fields to include in the HTTP response header.
```

Comment:
was this file referenced from somewhere like index.html?

Reply:
no